src/Interface.js

```diff
@@ -288,18 +296,18 @@ export function declaration (name, value, instance) {
  * @param {string} name
  * @param {string} value
  * @param {object} instance
- * @param {object} handler
- * @param {object} handlers
+ * @param {object} current
+ * @param {object} state
  */
-export function event (name, value, instance, handler, handlers) {
-	if (handlers) {
-		if (handlers[name] === undefined) {
-			instance.addEventListener(name, handler, false)
+export function event (name, value, instance, current, state) {
+	try {
```

Comment:
why swallow errors here?

Reply:
We probably should catch the error and retry with `{passive: true}` as the last argument if it throws per google chrome's "non-passive event listener" exceptions of late.